docs/source/entities-advanced.mdx

```diff
@@ -57,6 +57,99 @@ type Organization {
 }
 ```
 
+### Differing `@key`s across subgraphs
+
+An entity often has the exact same `@key` field(s) across subgraphs, but this isn't required. For example, you can define a `Product` entity shared between subgraphs, one with `sku` and `upc` as the `@key`s and one with only `upc` as the `@key` field:
+
+<CodeColumns>
+
+```graphql title="Products subgraph"
+type Product @key(fields: "sku") @key(fields: "upc") {
+  sku: ID!
+  upc: String!
+  name: String!
+  price: Int
+}
+```
+
+```graphql title="Inventory subgraph"
+type Product @key(fields: "upc") {
+  upc: String!
+  inStock: Boolean!
+}
+```
+
+</CodeColumns>
+
+To merge entities between subgraphs, the entity must have at least one shared field between subgraphs. For example, you can't merge the `Product` entity defined in the following subgraphs because they don't share any fields:
```

Comment:
maybe something along those lines
```suggestion
To merge entities between subgraphs, the entity must have at least one shared field between subgraphs. For example, you can't merge the `Product` entity defined in the following subgraphs because they don't share any fields specified in `@key` selection set:
```